test/e2e/live_iso_test.go

```diff
@@ -22,6 +22,7 @@ func liveIsoTest() {
 	var (
 		liveISOImageURL = e2eConfig.GetVariable("LIVE_ISO_IMAGE")
 	)
+	allMachinesCount := 4
```

Comment:
Here I would suggest getting NUM_NODE variable from environment and use that.

Reply:
Good point